packages/service-clients/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -30,32 +28,26 @@ import { TinyliciousClient } from "../index.js";
 
 import { TestDataObject } from "./TestDataObject.js";
 
-const corruptedAliasOp = async (
-	runtime: IContainerRuntime,
-	alias: string,
-): Promise<boolean | Error> =>
-	new Promise<boolean>((resolve, reject) => {
-		runtime.once("dispose", () => reject(new Error("Runtime disposed")));
-		(runtime as any).submit(ContainerMessageType.Alias, { id: alias }, resolve);
```

Comment:
`processAliasMessages` internally invokes the `localOpMetadata` to resolve its own promise, which I assume is the only way this would have worked in the past?  Though I don't see how it could have gotten to that point before erroring out as a malformed op.  In any case, I don't think the test needs to await that resolution to remain valid.